tests/json.ml

```diff
@@ -5,13 +5,122 @@ let good = "
  { \"a\":\"b\" }
 "
 
-let _ =
-  Printf.printf "Parsing good JSON ... %!";
-  let _ = Jsonrpc.of_string good in
+let v1 = "{
```

Comment:
I suspect some interaction between camlp4 and `{| |}` strings. I don't understand why they are not working here. So keep the `"..."` strings.